document/PolicyComplianceProfile.mdk

```diff
@@ -1 +1,65 @@
-# Policy Compliance Profile { #policy-compliance-profile }
\ No newline at end of file
+# Policy Compliance Profile { #policy-compliance-profile }
```

Comment:
we haven't adequately settled the importance of completely documenting the analysis targets. as we discussed yesterday, I am increasingly of the opinion that we should remove all things related to policy and compliance from this document, preserve that information elsewhere, finalize the core format, then convene a special effort around compliance (where we loop in compliance experts).